contents/docs/self-host/deploy/digital-ocean.mdx

```diff
@@ -10,55 +10,79 @@ showTitle: true
+Alternatively use [doctl](https://github.com/digitalocean/doctl) which also sets up kubeconfig for you.
 
 ```console
 doctl kubernetes cluster create posthog-cluster --count=2 --size="s-2vcpu-4gb" --region=sfo3 --1-clicks=posthog
 ```
 
-### Accessing Posthog
+### Accessing PostHog
 
 After deploying PostHog to your Kubernetes cluster you can get the IP to connect to one of two ways:
+
 1. Navigate to the IP address of the load balancer created by Kubernetes using your web browser. You can grab this by using the DigitalOcean web console ([networking tab](https://cloud.digitalocean.com/networking/load_balancers)).
-2. If you have `kubectl` configured for the cluster that PostHog is deployed to then run these commands to get your installation location:
-```console
-export INGRESS_IP=$(kubectl get --namespace posthog ingress posthog -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
-echo "Visit http://$INGRESS_IP to use PostHog\!"
-```
+2. If you have `kubectl` installed and configured for the cluster that PostHog is deployed to then run these commands to get your installation location:
+    ```console
+    export INGRESS_IP=$(kubectl get --namespace posthog ingress posthog -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
+    echo "Your PostHog IP address is $INGRESS_IP"
+    echo "Visit http://$INGRESS_IP to use PostHog\!"
+    ```
 
-Congrats, you have a working Posthog instance =)
+Congrats, you have a working PostHog instance =)
+
+You can now [integrate your development applications](/docs/integrate) with your PostHog deployment for testing purposes.
+
+> Note: You will only be able to test web apps on HTTP such as those running on `localhost`.
 
 But there's one more important step before using it in production: making sure the data is not at risk, see the next section.
 
 ### Securing your 1-click install
 
-Sadly it's not possible to provide parameters to DigitalOcean yet, so we need post-install steps to enable TLS.
- 
+It's [not yet possible to provide parameters to DigitalOcean](https://github.com/digitalocean/marketplace-kubernetes/issues/230), so we need a post-install steps to enable TLS.
+
+> In order to do this you will need [kubectl](https://kubernetes.io/docs/tasks/tools/) and [helm](https://helm.sh/) installed.
+
 #### 1. Lookup external IP
 
-```console
-kubectl get svc --namespace posthog posthog-ingress-nginx-controller
-```
+As above, you can get the IP to connect to one of two ways:
+
+1. Navigate to the IP address of the load balancer created by Kubernetes using your web browser. You can grab this by using the DigitalOcean web console ([networking tab](https://cloud.digitalocean.com/networking/load_balancers)).
+2. If you have `kubectl` installed and configured for the cluster that PostHog is deployed to then run these commands to get your installation location:
+    ```console
+    export INGRESS_IP=$(kubectl get --namespace posthog ingress posthog -o jsonpath="{.status.loadBalancer.ingress[0].ip}")
```

Comment:
Why are we asking users to create a variable here? Is it referenced later?

Reply:
There's an `echo` statement on the next line that uses it. But it's not used beyond that. I believe the purpose here was just to add some textual context to the message.